README.md

```diff
@@ -2,37 +2,52 @@
 
 |             | Info
 |-------------|-------------------------------------------------
-| WHEN        | Saturday at 11:00 am ET 
+| WHEN        | TBD
 | WHERE       | GeoLatinas Zoom
-| CHAT       | #coding_group channel in GeoLatinas Slack
-| REQUIREMENTS | Be a GeoLatinas member!  
+| CHAT       | #coding-group in GeoLatinas Slack
+| REQUIREMENTS | Be a GeoLatinas member ->
```

Comment:
```suggestion
| REQUIREMENTS | [Be a GeoLatinas member](https://geolatinas.weebly.com/get-involved.html)
```

Reply:
Gracias @santisoler . Thanks for cleaning this up 💯

Reply:
Hola @santisoler rompi cosas. 😄  pero no aproposito. Cuando acepto tu suggestion y pulso "resolve" no se incluyen en el branch o si? Tengo que esperar hasta que tu hagas commit.. 
Quise mergear pensando que todos tu cambios pasarian y cuando veo en el main no se pasaron... entonces hice revert. Pero todavia existe el otro branch.

Reply:
jaja, si, vi que no comiteaste mis suggestions, sino que marcaste los comentarios como resueltos.

No te preocupes, ahora armo un PR para estos cambios. Una vez que el PR está mergeado, ya está, queda como archivo. Si hay algo más que cambiar, abrimos otro PR.